Tools/MonoGame.Content.Builder.Editor/MainWindow.cs

```diff
@@ -557,7 +544,7 @@ private void CmdDebugMode_Executed(object sender, EventArgs e)
 
         private void CmdHelp_Executed(object sender, EventArgs e)
         {
-            Process.Start("http://www.monogame.net/documentation/?page=Pipeline");
+            Process.Start(new ProcessStartInfo() { FileName = "https://docs.monogame.net/articles/tools/pipeline.html", UseShellExecute = true, Verb = "open" });
```

Comment:
@Jjagg - I wonder how we could appropriately deal with a link like this and have it point to the right version?  I guess look at the embedded EXE version and put it into the URL?

Reply:
I think just pointing to develop docs is best.

Reply:
We can easily read the version from the assembly version attribute, as long as we have docs setup correctly.

Reply:
Also I could really use this merged, I have a PR incoming with a questionable commit XD The URL can be fixed in a separate one.